curriculum/class-00/lab-a/starter-code/js/app.js

```diff
@@ -59,10 +59,10 @@ function handleClick(event) {
     showList();
     makeChart();
   }
-  for(var i = 0; i < names.length; i++){
+  for(const i = 0; i < names.length; i++){
```

Comment:
you want to use a `let` rather than a `const` in a for loop, otherwise the loop cant run